index.html

```diff
@@ -0,0 +1,240 @@
+                Repudiandae laboriosam a quis, enim. <span class="hashtag">#Smash</span>&#128512</p>
+          </div>
+          <div class="main__bottom-pic-2">
+            <img
+              class="avatar-three"
+              src="../boolean-uk-html-twitter/images/avatar-three.jpg"
+              alt="avatar-three"
+            />
+          </div>
+          <div class="main__bottom-text-2">
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit.
+              Repudiandae laboriosam a quis, enim. 
+            </p>
+            <p>Lorem ipsum <span class="hashtag">#Smash</span>&#128513</p>
+            <img
+              class="big-pic"
+              src="../boolean-uk-html-twitter/images/big-pic.jpg"
+              alt="big-pic"
+            />
+          </div>
+          <div class="main__bottom-pic-3">
+            <img
+              class="avatar-three"
+              src="../boolean-uk-html-twitter/images/avatar-three.jpg"
+              alt="avatar-three"
+            />
+          </div>
+          <div class="main__bottom-text-3">
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit.
+              Repudiandae laboriosam a quis, enim.
+            </p>
+            <p>Lorem ipsum <span class="hashtag">#Smash</span&#128512>&#128512</p>
+            <img
+              class="big-pic"
+              src="../boolean-uk-html-twitter/images/big-pic.jpg"
+              alt="big-pic"
+            />
+          </div>
+          <div class="main__bottom-pic-4">
+            <img
+              class="avatar-three"
+              src="../boolean-uk-html-twitter/images/avatar-three.jpg"
+              alt="avatar-three"
+            />
+          </div>
+          <div class="main__bottom-text-4">
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit.
+              Repudiandae laboriosam a quis, enim.
+            </p>
+            <p>Lorem ipsum <span class="hashtag">#Smash</span>&#128513</p>
+            <img
+              class="big-pic"
+              src="../boolean-uk-html-twitter/images/big-pic.jpg"
+              alt="big-pic"
+            />
+          </div>
+        </div>
+      </div>
+      <div class="lh-sb">
+        <nav class="nav-grid">
+            <div id="logo" ><a href="https://www.google.com/"><svg class="icon twitter" xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" viewBox="0 0 24 24"><path d="M22,5.8a8.49,8.49,0,0,1-2.36.64,4.13,4.13,0,0,0,1.81-2.27,8.21,8.21,0,0,1-2.61,1,4.1,4.1,0,0,0-7,3.74A11.64,11.64,0,0,1,3.39,4.62a4.16,4.16,0,0,0-.55,2.07A4.09,4.09,0,0,0,4.66,10.1,4.05,4.05,0,0,1,2.8,9.59v.05a4.1,4.1,0,0,0,3.3,4A3.93,3.93,0,0,1,5,13.81a4.9,4.9,0,0,1-.77-.07,4.11,4.11,0,0,0,3.83,2.84A8.22,8.22,0,0,1,3,18.34a7.93,7.93,0,0,1-1-.06,11.57,11.57,0,0,0,6.29,1.85A11.59,11.59,0,0,0,20,8.45c0-.17,0-.35,0-.53A8.43,8.43,0,0,0,22,5.8Z"/></svg></a></div>
```

Comment:
Would be nice if you used the `<img` tag to import the SVG here. That way we reduce the number of lines in our HTML and make it a little easier to read :)